src/internal/puck/constant-value-fields/CallToAction.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { CustomField, FieldLabel, AutoField } from "@measured/puck";
+import { CTAProps } from "../../../components/puck/atoms/cta.tsx";
+
+export const CTA_CONSTANT_CONFIG: CustomField<CTAProps> = {
+  type: "custom",
+  render: ({ onChange, value }) => {
+    return (
+      <>
+        <FieldLabel label={"Label"} className="ve-inline-block ve-pt-4">
```

Comment:
All of the const-value-fields are pretty repetitive. Maybe we should simplify them to just iterate over an object that defines the label and property name, then all of these components would basically be one-liners.

Reply:
some of these are more complicated

such as TextList which has buttons

or Text which just uses the TextField type

We could create a shared helper method for the others though